Packs/FlashpointFeed/Layouts/layoutscontainer-Flashpoint_Indicator.json

```diff
@@ -0,0 +1,386 @@
+{
```

Comment:
Please add the Relationships section to the layout

Reply:
sure @ChanochShayner, it seems it is recently released. Will do it anyway.